classes/utils/Config.py

```diff
@@ -18,18 +18,10 @@ class ConfigParser:
     :raises Exceptions.UnsupportedConfigType: if _type is not between 1 and 2
     """
 
-    def __init__(self, config_location, _type: int = 1):
+    def __init__(self, _type: int = 1):
+        config_location = 'server.properties'
```

Comment:
If anything we should end it with .yaml since that is what we are parsing it with, .properties might be a bit misleading